interfaces/builtin/tpm12.go

```diff
@@ -0,0 +1,118 @@
+package builtin
+
+import (
+	"bytes"
+
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var tpm12PermanentSlotAppArmor = []byte(`
+# Description: for tcsd
+# Usage: common
+
+#include <abstractions/apache2-common>
+#include <abstractions/base>
+#include <abstractions/postfix-common>
+
+capability dac_override,
+
+/dev/tpm0 rw,
+/etc/tcsd.conf r,
+/usr/sbin/tcsd mr,
+/var/lib/tpm/system.data rw,
```

Comment:
This shouldn't be used like that. Please make tpm tools respect $SNAP and $SNAP_DATA